Makefile

```diff
@@ -46,13 +47,16 @@ $(LOCAL_BRIDGE_PLUGIN_BINARY): $(SOURCES)
 generate: $(SOURCES)
 	go generate -x ./pkg/... ./plugins/...
 
-.PHONY: unit-test integration-test
+.PHONY: unit-test integration-test e2e-test
 unit-test: $(SOURCES)
 	go test -v -cover -race -timeout 10s ./pkg/... ./plugins/...
 
 integration-test: $(SOURCE)
 	go test -v -tags integration -race -timeout 10s ./pkg/... ./plugins/...
 
+e2e-test:  $(SOURCE) plugins
+	sudo -E CNI_PATH=${ROOT}/bin/plugins ${GO_EXECUTABLE} test -v -tags e2e -race -timeout 60s ./plugins/...
```

Comment:
I think (at least on Amazon Linux) that we've got a value compiled in for the `secure_path` sudoers setting, and that's overriding PATH even when `-E` is in effect. So yeah, stick with `${GO_EXECUTABLE}`